src/app/shared/document/document-detail/document-detail.component.ts

```diff
@@ -36,6 +36,9 @@ import {UiRow} from '../../../core/ui/ui-row';
 import DeleteConfirm = DocumentsAction.DeleteConfirm;
 import {Perspective, perspectivesMap} from '../../../view/perspectives/perspective';
 import {PerspectiveService} from '../../../core/perspective.service';
+import {LinkInstancesAction} from '../../../core/store/link-instances/link-instances.action';
+import {selectLinkTypesByDocumentId} from '../../../core/store/link-types/link-types.state';
+import {selectLinkInstancesByTypeAndDocuments} from '../../../core/store/link-instances/link-instances.state';
```

Comment:
I assume these newly added imports are not used anywhere in this component since there are no related changes.